packages/payment-detection/src/any/any-to-any-proxy-info-retriever.ts

```diff
@@ -25,60 +16,51 @@ type TransferWithConversionAndReferenceArgs = {
 
 /** TransferWithReferenceAndFee event */
 type TransferWithReferenceAndFeeArgs = {
-  tokenAddress: string;
+  tokenAddress?: string;
```

Comment:
Signature of TransferWithReferenceAndFee is different between eth-fee-proxy and erc20-fee-proxy